.github/workflows/build-scan-push.yml

```diff
@@ -81,7 +81,10 @@ jobs:
           image: ${{steps.setImageDetails.outputs.fullImageName}}
           fail-build: true
           severity-cutoff: "medium"
-          only-fixed: true
```

Comment:
Wondering why you are removing "only-fixed"?

Reply:
Thought it would be more valuable to be alerted of issues and then add them to exclude file.  Rather than silently ignore them.

Reply:
Sure, that's fine, if you are happy to keep the exclude file up to date 😁. Before we had this setting in Node it became a chore, but in an ideal world I think you are right. Happy to approve and see how we go.

Reply:
Watch out for a PR next week to put it back!